core/src/main/java/org/mobilitydata/gtfsvalidator/validator/ValidationContext.java

```diff
@@ -48,14 +49,42 @@ public static Builder builder() {
    *   <li>unit tests may need to override the current time.
    * </ul>
    *
-   * @return The time when validation started as @code{ZonedDateTime}
+   * @return The time when validation started as {@code CurrentDateTime}
+   */
+  public abstract CurrentDateTime currentDateTime();
```

Comment:
Now we have both currentDateTime() and now() methods. We should have only one. I see the following alternatives.

1. Remove changes to ValidationContext from this PR.
2. Remove now() and update all callers of that method.

Please also update the title and description of this PR.

Reply:
Removed changes to ValidationContext from this PR and updated the PR title and description.